apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -43,4 +44,47 @@ const Template = ({ type }: InputProps) => html`
     <pie-input type=${ifDefined(type)}></pie-input>
 `;
 
+const FormTemplate: TemplateFunction<InputProps> = (props: InputProps) => html`
+<p id="formLog" style="display: none; font-size: 2rem; color: var(--dt-color-support-positive);"></p>
+<h2>Fake form</h2>
+<form id="testForm">
+        <p>Required fields are followed by <strong><span aria-label="required">*</span></strong>.</p>
```

Comment:
Nit: I like having clear feedback on what is happening, but I am not sure about it being displayed before everything else.